Can not create SQL server

Getting exception when creating SQL server in a new resource group

```
java.lang.RuntimeException: com.microsoft.azuretools.adauth.AuthException: AD Auth token endpoint returned HTTP status code 400. Error info: {"error":"invalid_request","error_description":"AADSTS90002: Tenant 'null' not found. This may happen if there are no active subscriptions for the tenant. Check with your subscription administrator.\r\nTrace ID: 67300\r\nCorrelation ID: ecb2b79f-f4e7-4cf0-a4eb-1e778339c04d\r\nTimestamp: 2019-08-05 19:10:17Z","error_codes":[90002],"timestamp":"2019-08-05 19:10:17Z","trace_id":"6fd900","correlation_id":"ecb2c04d"}
	at rx.exceptions.Exceptions.propagate(Exceptions.java:57)
	at rx.observables.BlockingObservable.blockForSingle(BlockingObservable.java:463)
	at rx.observables.BlockingObservable.single(BlockingObservable.java:340)
	at com.microsoft.azure.management.sql.implementation.ServersInner.list(ServersInner.java:208)
	at com.microsoft.azure.management.resources.fluentcore.arm.collection.implementation.TopLevelModifiableResourcesImpl.list(TopLevelModifiableResourcesImpl.java:118)
	at com.microsoft.azuretools.core.mvp.model.database.AzureSqlServerMvpModel.getSqlServersBySubscriptionId(AzureSqlServerMvpModel.kt:142)
	at com.microsoft.azuretools.core.mvp.model.database.AzureSqlServerMvpModel.listSqlServersBySubscriptionId(AzureSqlServerMvpModel.kt:62)
	at com.microsoft.azuretools.core.mvp.model.database.AzureSqlServerMvpModel.refreshSubscriptionToSqlServerMap(AzureSqlServerMvpModel.kt:46)
	at com.microsoft.azuretools.core.mvp.model.database.AzureSqlDatabaseMvpModel.refreshSqlServerToSqlDatabaseMap(AzureSqlDatabaseMvpModel.kt:49)
	at com.microsoft.intellij.ui.forms.sqlserver.CreateSqlServerDialog$updateAzureModelInBackground$1.run(CreateSqlServerDialog.kt:310)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:894)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:169)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:591)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:537)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:156)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:408)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:294)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: com.microsoft.azuretools.adauth.AuthException: AD Auth token endpoint returned HTTP status code 400. Error info: {"error":"invalid_request","error_description":"AADSTS90002: Tenant 'null' not found. This may happen if there are no active subscriptions for the tenant. Check with your subscription administrator.\r\nTrace ID: 6fd5ddb654b7300\r\nCorrelation ID: ec4d\r\nTimestamp: 2019-08-05 19:10:17Z","error_codes":[90002],"timestamp":"2019-08-05 19:10:17Z","trace_id":"6fd90","correlation_id":"ed"}
	at com.microsoft.azuretools.adauth.AuthContext.sendRequest(AuthContext.java:260)
	at com.microsoft.azuretools.adauth.AuthContext.access$100(AuthContext.java:46)
	at com.microsoft.azuretools.adauth.AuthContext$1.getTokenWithRefreshToken(AuthContext.java:245)
	at com.microsoft.azuretools.adauth.CacheDriver.acquireTokenFromMrrt(CacheDriver.java:117)
	at com.microsoft.azuretools.adauth.CacheDriver.refreshEntryIfNecessary(CacheDriver.java:109)
	at com.microsoft.azuretools.adauth.CacheDriver.find(CacheDriver.java:55)
	at com.microsoft.azuretools.adauth.AuthContext.acquireTokenFromCache(AuthContext.java:213)
	at com.microsoft.azuretools.adauth.AuthContext.acquireToken(AuthContext.java:126)
	at com.microsoft.azuretools.authmanage.AdAuthManager.getAccessToken(AdAuthManager.java:73)
	at com.microsoft.azuretools.authmanage.RefreshableTokenCredentials.getToken(RefreshableTokenCredentials.java:65)
	at com.microsoft.azure.credentials.AzureTokenCredentials.getToken(AzureTokenCredentials.java:74)
	at com.microsoft.azure.credentials.AzureTokenCredentialsInterceptor.intercept(AzureTokenCredentialsInterceptor.java:36)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at com.microsoft.azure.management.resources.fluentcore.utils.ResourceManagerThrottlingInterceptor.intercept(ResourceManagerThrottlingInterceptor.java:54)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at com.microsoft.azure.management.resources.fluentcore.utils.ProviderRegistrationInterceptor.intercept(ProviderRegistrationInterceptor.java:43)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at com.microsoft.azuretools.telemetry.TelemetryInterceptor.intercept(TelemetryInterceptor.java:46)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at com.microsoft.rest.interceptors.BaseUrlHandler.intercept(BaseUrlHandler.java:43)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at com.microsoft.rest.interceptors.RequestIdHeaderInterceptor.intercept(RequestIdHeaderInterceptor.java:29)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:254)
	at okhttp3.RealCall.execute(RealCall.java:92)
	at retrofit2.OkHttpCall.execute(OkHttpCall.java:186)
	at retrofit2.adapter.rxjava.CallExecuteOnSubscribe.call(CallExecuteOnSubscribe.java:40)
	at retrofit2.adapter.rxjava.CallExecuteOnSubscribe.call(CallExecuteOnSubscribe.java:24)
	at rx.Observable.unsafeSubscribe(Observable.java:10327)
	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:48)
	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:33)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:48)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:30)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:48)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:30)
	at rx.Observable.subscribe(Observable.java:10423)
	at rx.Observable.subscribe(Observable.java:10390)
	at rx.observables.BlockingObservable.blockForSingle(BlockingObservable.java:443)
	... 21 more
Caused by: java.io.IOException: AD Auth token endpoint returned HTTP status code 400. Error info: {"error":"invalid_request","error_description":"AADSTS90002: Tenant 'null' not found. This may happen if there are no active subscriptions for the tenant. Check with your subscription administrator.\r\nTrace ID: 6fd90\r\nCorrelation ID: ecbd\r\nTimestamp: 2019-08-05 19:10:17Z","error_codes":[90002],"timestamp":"2019-08-05 19:10:17Z","trace_id":"6fd00","correlation_id":"ecb4d"}
	at com.microsoft.azuretools.adauth.HttpHelper.sendPostRequestAndDeserializeJsonResponse(HttpHelper.java:92)
	at com.microsoft.azuretools.adauth.AuthContext.sendRequest(AuthContext.java:256)
	... 64 more
```

--- a/PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/ui/forms/sqlserver/CreateSqlServerDialog.kt
+++ b/PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/ui/forms/sqlserver/CreateSqlServerDialog.kt
@@ -182,7 +182,10 @@ class CreateSqlServerDialog(private val lifetimeDef: LifetimeDefinition,
                         sqlServerName,
                         cbLocation.getSelectedValue()!!.region(),
                         pnlResourceGroup.rdoCreateNew.isSelected,
-                        pnlResourceGroup.cbResourceGroup.getSelectedValue()!!.name(),
+                        when {
+                            pnlResourceGroup.rdoCreateNew.isSelected -> pnlResourceGroup.txtResourceGroupName.text
+                            else -> pnlResourceGroup.cbResourceGroup.getSelectedValue()!!.name()
+                        },
                         txtAdminLoginValue.text,
                         passAdminPassword.password)
             }

--- a/Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/database/AzureSqlServerMvpModel.kt
+++ b/Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/database/AzureSqlServerMvpModel.kt
@@ -39,8 +39,7 @@ object AzureSqlServerMvpModel {
     private val subscriptionIdToSqlServersMap = ConcurrentHashMap<String, List<ResourceEx<SqlServer>>>()
 
     fun refreshSubscriptionToSqlServerMap() {
-        val azureManager = AuthMethodManager.getInstance().azureManager ?: return
-        val subscriptions = azureManager.subscriptions
+        val subscriptions = AzureMvpModel.getInstance().selectedSubscriptions
 
         for (subscription in subscriptions) {
             listSqlServersBySubscriptionId(subscription.subscriptionId(), true)

